### PR TITLE
MAYA-108919: Center pivot command support test.

### PR DESCRIPTION
**Pull request by HamedSabri-adsk**
Adding a unit test around the "Center Pivot" command support.

### CODE REVIEW

File: test/lib/ufe/testComboCmd.py [hunk 1]
```diff
@@ -805,6 +805,55 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
+    @unittest.skipIf(mayaUtils.previewReleaseVersion() < 124, 'Center point command is only available in Maya Preview Release 124 or later.')
```
Comment (HamedSabri-adsk) [inline test/lib/ufe/testComboCmd.py:808]: @ppt-adsk  for now I set the PR version to 124.  We might want to change this or come up with another way to detect my changes on Maya side.

File: test/lib/ufe/testComboCmd.py [hunk 2]
```diff
@@ -805,6 +805,55 @@ def testFallback(self):
...
+        import mayaUsd_createStageWithNewLayer
+        mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+        proxyShapePath = ufe.PathString.path('|stage1|stageShape1')
+        proxyShapeItem = ufe.Hierarchy.createItem(proxyShapePath)
+        proxyShapeContextOps = ufe.ContextOps.contextOps(proxyShapeItem)
+        proxyShapeContextOps.doOp(['Add New Prim', 'Capsule'])
+
+        capsulePath = ufe.PathString.path('|stage1|stageShape1,/Capsule1')
+        capsuleItem = ufe.Hierarchy.createItem(capsulePath)
+        capsulePrim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(capsulePath))
```
Comment (ppt-adsk) [inline test/lib/ufe/testComboCmd.py:822]: Unused.

**Review by ppt-adsk — COMMENTED** [review]
Thanks for the test!

File: test/lib/ufe/testComboCmd.py (hunk 2 above)
Comment (HamedSabri-adsk) [reply test/lib/ufe/testComboCmd.py]: c535ddea35518be76cddaeae323060203859eec0

**Review by ppt-adsk — APPROVED** [review]

File: test/lib/ufe/testComboCmd.py [hunk 3]
```diff
@@ -805,6 +805,54 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
+    @unittest.skipIf(int(cmds.about(majorVersion=True)) > 2022, 'Center pivot command is only available in Maya Preview Release 124 or later.')
```
Comment (kxl-adsk) [inline test/lib/ufe/testComboCmd.py:808]: We want this test to be run in master builds, not skipped in master builds. Additionally, we can leverage API version to detect update versions. Here is a check that will do this:
```suggestion
    @unittest.skipIf(int(cmds.about(apiVersion=True)) <= 20220000, 'Center pivot command is only available in Maya Preview Release 124 or later.')
```

We are still lacking coverage in PFs, this is why it wasn't detected when you run it.

**Review by kxl-adsk — CHANGES_REQUESTED** [review]
Combo test is failing with PR122. See my comments in the code for more details.

File: test/lib/ufe/testComboCmd.py [hunk 4]
```diff
@@ -805,7 +805,7 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
-    @unittest.skipIf(mayaUtils.previewReleaseVersion() < 124, 'Center pivot command is only available in Maya Preview Release 124 or later.')
+    @unittest.skipIf(int(cmds.about(apiVersion=True)) <= 20220000, 'Center pivot command is only available in Maya Preview Release 124 or later.')
```
Comment (ppt-adsk) [inline test/lib/ufe/testComboCmd.py:808]: Bit awkward that the reason for the skip doesn't match the conditional

**Review by kxl-adsk — APPROVED** [review]

File: test/lib/ufe/testComboCmd.py (hunk 4 above)
Comment (kxl-adsk) [reply test/lib/ufe/testComboCmd.py:808]: Agreed. @HamedSabri-adsk Please adapt the message.

**Pull request merged**